instat/dlgOneWayFrequencies.vb

```diff
@@ -21,8 +21,7 @@ Public Class dlgOneWayFrequencies
     Private bReset As Boolean = True
     Private bResetSubdialog As Boolean = False
     Private clsSjTab As New RFunction
-    Private clsSjPlot, clsPlotGrid, clsSelect As New RFunction
-    Private clsTableBaseOperator, clsGraphBaseOperator As New ROperator
+    Private clsSjPlot, clsPlotGrid As New RFunction

```

Comment:
I've reverted this back to functions and removed the operators. I didn't see any need for the select function. Instead we just get the data parameter from the receiver with the selected columns. If you think you do need it still then you don't have to merge this yet.